### PR TITLE
[fix]: mobile responsiveness and optimized images

### PR DESCRIPTION
**Pull request by grand-rick001**
@mog-rn Added mobile responsiveness
Mobile responsive

### CODE REVIEW

File: src/components/Landing/CallToAction.tsx
Comment (mog-rn) [inline src/components/Landing/CallToAction.tsx:1]: can you align the text to be at the start using the items-start class for tailwindcss in the particular div.

File: src/components/Landing/CallToAction.tsx [hunk 1]
```diff
@@ -123,20 +123,21 @@ const CallToAction = () => {
...
-									<ul>
-										<h2>
-											Improve efficiency and productivity
-											with our technology
-										</h2>
+									<h2>
+										Improve efficiency and productivity
+										with our technology
+									</h2>
+									<ul className="icon-list">
```
Comment (mog-rn) [inline src/components/Landing/CallToAction.tsx:139]: you can do the items-start in this icon-list class

Comment (mog-rn) [reply src/components/Landing/CallToAction.tsx:139]: Is this normal css class or a tailwindcss class

**Review by mog-rn — COMMENTED** [review]
@grand-rick001 you can review your work please with the changes I requested and if possible please attach the images of the final results.

File: src/components/Landing/CallToAction.tsx (hunk 1 above)
Comment (grand-rick001) [reply src/components/Landing/CallToAction.tsx:139]: It's plain css

**Review by mog-rn — APPROVED** [review]
@grand-rick001 will merge changes from this pull request

**Pull request merged**